vllm/entrypoints/openai/tool_parsers/utils.py

```diff
@@ -85,3 +91,30 @@ def find_all_indices(string, substring):
             break
         indices.append(index)
     return indices
+
+
+# partial_json_parser doesn't support extra data and
+# JSONDecorder.raw_decode doesn't support partial JSON
+def partial_json_loads(input_str, flags):
```

Comment:
Could you add typing to these functions? (sorry I know they were just moved in this PR!)